tests/test_bookmarks.py

```diff
@@ -56,6 +68,18 @@ def test_run(self):
         For EACH stream that is incrementally replicated there are multiple rows of data with
             different values for the replication key
         """
+        # Ensure tested streams have a record count which exceeds the API LIMIT
+        expected_records_1 = {x: [] for x in self.expected_streams()}
+        for stream in self.testable_streams():
+            existing_objects = self.client.get_all(stream, self.START_DATE)
+
+            if len(existing_objects) == 0:
+                # TODO change from 'failure you need data' to 'create data then so we can test'
```

Comment:
Neither of these messages appear here 😅 

Could the assertion message include the stream name?

Reply:
Added stream name. Creates aren't working for all streams yet and I haven't hit this point of missing data. Once one of those things happen I'll put this logic in.